include/z64actor.h

```diff
@@ -171,12 +171,12 @@ typedef struct Actor {
     /* 0x085 */ u8 floorBgId; // Bg ID of the floor polygon directly below the actor
     /* 0x086 */ s16 wallYaw; // Y rotation of the wall polygon the actor is touching
     /* 0x088 */ f32 floorHeight; // Y position of the floor polygon directly below the actor
-    /* 0x08C */ f32 yDistToWater; // Distance to the surface of active waterbox. Negative value means above water
+    /* 0x08C */ f32 yDirDistToWater; // Distance to the surface of active waterbox. Negative value means above water
     /* 0x090 */ u16 bgCheckFlags; // See comments below actor struct for wip docs. TODO: macros for these flags
     /* 0x092 */ s16 yawTowardsPlayer; // Y rotation difference between the actor and the player
     /* 0x094 */ f32 xyzDistToPlayerSq; // Squared distance between the actor and the player in the x,y,z axis
     /* 0x098 */ f32 xzDistToPlayer; // Distance between the actor and the player in the XZ plane
-    /* 0x09C */ f32 yDistToPlayer; // Dist is negative if the actor is above the player
+    /* 0x09C */ f32 yDirDistToPlayer; // Dist is negative if the actor is above the player
```

Comment:
```suggestion
    /* 0x09C */ f32 yDirDistToPlayer; // Directional Distance is negative if the actor is above the player
```
Same here

Reply:
```suggestion
    /* 0x09C */ f32 yDirDistToPlayer; // Directed Distance is negative if the player is below.
```

Reply:
If there was a previous conversation on this, sorry for missing it. I think I do like `depthIntoWater` and `dirDistBelowPlayer` a little more than the new names (although I'm fine with the new names over the old names). Not as strong opinion though

Reply:
It's an archived thread in \#mm-names-discussion. This new set of names is sort of where the conversation ended. I personally do not like depth as much- it's harder for me to think about what sign means what. My ideal would have been yVecTo, but that was shot down pretty quick for good reason. yDirDistTo is almost as good, just a bit wordier, but equally as clear, imo.